### DISCUSSION TITLE
Add support for chaning ones identity (badges)

### DISCUSSION THREAD

**Discussion by robsdedude** [Chatterino/chatterino2#4068] [Ideas] [answered]
The native Twtich chat allows you to change your badges, e.g., choose "Watching without audio", hide your badge flair and more. I coulnd't find this feature in Chatterino. I think it would be great to have especially with the somewhat recently "no audio", "no video" badges Twitch added.

**Comment by Felanbird** [comment, marked as answer]
We can't do that, there's no API for switching badges.

**Reply by robsdedude** [reply]
What a shame, Twitch :(

**Reply by robsdedude** [reply]
Double checked the API docs, nothing there, as expected. Inspected the web page, using a non-public graphql API. Twitch be Twitch 😞
